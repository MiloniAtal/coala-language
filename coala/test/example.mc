void main() {
    int x;
    x = 1;
    string s;
    s = "c";
    string x;
    string x = "ab";
    string s = "ab";
    return;
}


