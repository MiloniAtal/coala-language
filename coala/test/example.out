Parsed program: 


int main()
{
int x;
string s;
s = "coala";
x = 5;
print(x);
return x;
}
